src/priorityqueue.jl

```diff
@@ -175,6 +175,17 @@ function get(pq::PriorityQueue{K,V}, key, deflt) where {K,V}
     i == 0 ? deflt : pq.xs[i].second
 end
 
+function get!(pq::PriorityQueue{K,V}, key, deflt) where {K,V}
```

Comment:
please spell out `deflt` fully,
it isn't much longer, and it makes it more readable.
(I spent a few seconds thinking `deflate`, wat?)

I am guessing you just copies the one above, which is fair.
It would be really great if you could fix that one on the way past.
(Some of this code is older than the idea of coding standard in julia)
Also nio need to mention `K` or `V`.

Reply:
@oxinabox Agreed! I just did a cut and paste to match the old code. I will fix both places.